README.md

```diff
@@ -0,0 +1,6 @@
+# curriculum-cr-training-mobile-menu
+
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
```

Comment:
- [ ] Kindly use the Microverse README template for the [README file](https://github.com/microverseinc/readme-template) with its styling and add the same information you added to that template.